ntbs-service/Pages/Shared/_NotificationLayout.cshtml

```diff
@@ -1,46 +1,64 @@
-            </li>
-            <li class="@("contents-list-item " + (currentPage == "ContactTracing" ? "contents-list-item--current" : ""))">
-                <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/Notifications/Edit/ContactTracing?id=@Model.Notification.NotificationId&isBeingSubmitted=@Model.Notification.ShouldValidateFull">Contact Tracing</a>
-            </li>
-            <li class="@("contents-list-item " + (currentPage == "SocialRiskFactors" ? "contents-list-item--current" : ""))">
-                <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/Notifications/Edit/SocialRiskFactors?id=@Model.Notification.NotificationId&isBeingSubmitted=@Model.Notification.ShouldValidateFull">Social Risk Factors</a>
-            </li>
-            <li class="@("contents-list-item " + (currentPage == "Travel" ? "contents-list-item--current" : ""))">
-                <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/Notifications/Edit/Travel?id=@Model.Notification.NotificationId&isBeingSubmitted=@Model.Notification.ShouldValidateFull">Travel/visitor history</a>
-            </li>
-            <li class="@("contents-list-item " + (currentPage == "Comorbidities" ? "contents-list-item--current" : ""))">
-                <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/Notifications/Edit/Comorbidities?id=@Model.Notification.NotificationId&isBeingSubmitted=@Model.Notification.ShouldValidateFull">Co-morbidities</a>
-            </li>
-            <li class="@("contents-list-item " + (currentPage == "Immunosupression" ? "contents-list-item--current" : ""))">
-                <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/Notifications/Edit/Immunosupression?id=@Model.Notification.NotificationId&isBeingSubmitted=@Model.Notification.ShouldValidateFull">Immunosupression</a>
-            </li>
-            <li class="@("contents-list-item " + (currentPage == "PreviousHistory" ? "contents-list-item--current" : ""))">
-                <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/Notifications/Edit/PreviousHistory?id=@Model.Notification.NotificationId&isBeingSubmitted=@Model.Notification.ShouldValidateFull">Previous History</a>
-            </li>
-        </div>
-    </nhs-grid-column>
-    <nhs-grid-column grid-column-width="ThreeQuarters">
+    @if (!(bool)ViewData["IsLinkedNotificationsPage"])
+    {
+        <nhs-grid-column grid-column-width="OneQuarter">
+            <div>
+                <li class="@("contents-list-item " + ((bool)ViewData["IsPatientPage"] ? "contents-list-item--current" : ""))">
+                    <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="@Model.Notification.PatientEditPath">Person details</a>

```

Comment:
Calling a property on notification now rather than writing out the full link here